src/commissaire_service/service/__init__.py

```diff
@@ -153,10 +154,32 @@ def __init__(self, exchange_name, connection_url, qkwargs):
         :type connection_url: str
         :param qkwargs: One or more dicts keyword arguments for queue creation
         :type qkwargs: list
+        :param config_files: Paths to the configuration file locations.
+        :type config_files: tuple(str, str)
```

Comment:
This config files tuple will work, but it strikes me as kinda ugly.

How about instead we track the default config file path as a class attribute (named `DEFAULT_CONFIGURATION_FILE` or some such).  In the base `CommissaireService` class it can be assigned `C.DEFAULT_CONFIGURATION_FILE`.  Subclasses would override it to `'/etc/commissaire/{SERVICE_NAME}.conf'`.

Then the config file argument to `CommissaireService.__init__` can still be an optional string value from the `--config-file` option (`config_file=None`), like what the subclasses take.  You can then pass the argument and the class attribute straight to `read_config_file()`.

Make sense?  That seems cleaner to me.